src/main/java/model/Cannon.java

```diff
@@ -0,0 +1,93 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cannon extends Piece{
```

Comment:
```suggestion
public class Cannon extends Piece {
```

인텔리제이에 자동정렬기능을 사용하면 이런 컨밴션들은 자동으로 맞출 수 있습니다 :)

Reply:
감사합니다!